profiles/native_vk.yaml

```diff
@@ -0,0 +1,9 @@
+# This file was auto generated, take caution if manually editing.
+plugins: audio_pipeline,offline_cam,offline_imu,gtsam_integrator,pose_prediction,display_vk,timewarp_vk,vkdemo,native_renderer,debugview
```

Comment:
There are dependency issues with this loading order.
`pose_prediction,display_vk,timewarp_vk,vkdemo,audio_pipeline,debugview,gtsam_integrator,offline_cam,offline_imu,native_renderer` worked for me

Reply:
Additionally, the automatically generated illixr.yaml seems to point to the wrong path for `demo_data`. Either the path is wrong or the folder is not copied.

Reply:
I fixed the plugin order in native_vk.
The demo data are currently not installed but I did fix where the demo_data entry is pointing